client/dashboard/task-list/tasks/payments/index.js

```diff
@@ -97,9 +97,48 @@ class Payments extends Component {
 		getHistory().push( getNewPath( {}, '/', {} ) );
 	}
 
+	isStripeEnabledByDefault() {
+		const { countryCode } = this.props;
+		// Stripe should be checked by default in the following countries: https://stripe.com/global.
+		const supportedCountries = [
```

Comment:
Good suggestion.

There is no public method for this like there is for eu countries, however I went ahead and added a patch to core that adds a method like this, so it can be the one source of truth:
https://github.com/woocommerce/woocommerce/pull/24897. This PR contains a fallback until that is released.

It also looks like core was missing 10 new countries. There are a few more pending, so it probably does change at least once a year or so as Stripe continues to roll out.

Reply:
I also hide the Stripe method if its not supported, which matches the current onboarding experience as well.

Reply:
That PR won't make it, so we'll just hardcode our own list. Updated the PR.